crystal_diffusion/callbacks/sampling_callback.py

```diff
@@ -222,22 +225,32 @@ def on_validation_epoch_end(self, trainer: Trainer, pl_model: LightningModule) -
         pc_sampler, unit_cell = self._create_sampler(pl_model)
 
         logger.info("Draw samples")
```

Comment:
The sampling code below is becoming involved. It would be great to wrap it into a `staticmethod` and test it.